frontend/app/routes/protected/multi-channel/send-validation.tsx

```diff
@@ -8,22 +8,18 @@ import { useTranslation } from 'react-i18next';
 import type { Info, Route } from './+types/send-validation';
 
 import { caseApi } from '~/.server/domain/multi-channel/services/case-api-service';
-import { getLocalizedApplicantGenderById } from '~/.server/domain/person-case/services/applicant-gender-service';
-import { getLocalizedApplicantSecondaryDocumentChoiceById } from '~/.server/domain/person-case/services/applicant-secondary-document-service';
-import { getLocalizedLanguageOfCorrespondenceById } from '~/.server/domain/person-case/services/language-correspondence-service';
-import { serverEnvironment } from '~/.server/environment';
-import { getLocalizedCountryById } from '~/.server/shared/services/country-service';
-import { getLocalizedProvinceById } from '~/.server/shared/services/province-service';
 import { requireAllRoles } from '~/.server/utils/auth-utils';
 import { i18nRedirect } from '~/.server/utils/route-utils';
 import { Button } from '~/components/button';
 import { ContextualAlert } from '~/components/contextual-alert';
+import { InlineLink } from '~/components/links';
 import { PageTitle } from '~/components/page-title';
 import { AppError } from '~/errors/app-error';
 import { ErrorCodes } from '~/errors/error-codes';
 import { getTranslation } from '~/i18n-config.server';
 import { handle as parentHandle } from '~/routes/protected/person-case/layout';
-import { SinApplication } from '~/routes/protected/sin-application';
+import { ApplicationReview } from '~/routes/protected/sin-application/application-review';
+import { formatSinApplication } from '~/routes/protected/sin-application/validation.server';
```

Comment:
I don't think the `multi-channel` route(s) should be referencing anything from other routes. This creates couplings that will make future maintenance difficult.